### PR TITLE
Yonatan

### PR DESCRIPTION
**Pull request by Yonatane12345**

### CODE REVIEW

File: face_detection/12.Y34r5.4.5l4v3/Faces/embeddings.npy
Comment (natexcvi) [inline face_detection/12.Y34r5.4.5l4v3/Faces/embeddings.npy:1]: Remove this file

File: face_detection/__pycache__/cluster_funcs.cpython-311.pyc
Comment (natexcvi) [inline face_detection/__pycache__/cluster_funcs.cpython-311.pyc:1]: Remove this file

File: face_detection/__pycache__/face_embedding.cpython-311.pyc
Comment (natexcvi) [inline face_detection/__pycache__/face_embedding.cpython-311.pyc:1]: Remove this file

File: face_detection/blaze_face_short_range.tflite
Comment (natexcvi) [inline face_detection/blaze_face_short_range.tflite:1]: Remove this file

File: face_detection/cluster_funcs.py [hunk 1]
```diff
@@ -200,14 +249,20 @@ def cluster(embeddings):
 
 def split_images(labels,faces_path):
     opened_labels = {}
-    os.mkdir(os.path.join(faces_path,"clustered"))
+    clusters_path = os.path.join(faces_path,"clustered")
+    if os.path.exists(clusters_path):
+        shutil.rmtree(clusters_path)
+        time.sleep(0.5)
```
Comment (natexcvi) [inline face_detection/cluster_funcs.py:255]: Why do we need the sleep here?

File: gcp-vscode-ssh
Comment (natexcvi) [inline gcp-vscode-ssh:1]: What is this?

File: script_segmentation/script_segmentation.py [hunk 2]
```diff
@@ -181,8 +72,8 @@ def clean_text(filename):
     for x in text:
         tx = x.replace('\t', ' ').lstrip(" ")
         tx = re.sub(r'^\d+\n', r'', tx)
-        #tx = re.sub(r'\(.*\)', r'', tx) #TODO aborted
-        #tx = re.sub(r'\#\d+', r'', tx) #TODO aborted
+        #tx = re.sub(r'\(.*\)', r'', tx)
+        #tx = re.sub(r'\#\d+', r'', tx)
```
Comment (natexcvi) [inline script_segmentation/script_segmentation.py:76]: Remove commented out code

**Review by natexcvi — CHANGES_REQUESTED** [review]

File: face_detection/cluster_funcs.py (hunk 1 above)
Comment (Yonatane12345) [reply face_detection/cluster_funcs.py:255]: to make sure dir deleted

**Review by natexcvi — APPROVED** [review]

**Pull request merged**